irohad/simulator/impl/simulator.hpp

```diff
@@ -6,30 +6,32 @@
 #ifndef IROHA_SIMULATOR_HPP
 #define IROHA_SIMULATOR_HPP
 
-#include <boost/optional.hpp>
+#include "simulator/block_creator.hpp"
+#include "simulator/verified_proposal_creator.hpp"
```

Comment:
looks like those two includes should go after all other includes

Reply:
is there any reason to put them before anything else?

Reply:
>Use standard order for readability and to avoid hidden dependencies: Related header, C library, C++ library, other libraries' .h, your project's .h.

These two headers are "related headers" because the class inherits from `BlockCreator` and `VerifiedProposalCreator`.